### PR TITLE
Changes for Multi Environment Support and Streamer Refactor

### PR DESCRIPTION
**Pull request by sidddutta**
This contains the following changes:
1. Changes for multi-environment
2. Code refactor

### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### CODE REVIEW

File: jazz_cloud-logs-streamer/components/utils.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+
+module.exports = () => {
+    return {
+        extractJson: extractJson,
+        isNumeric: isNumeric,
+        isValidJson: isValidJson,
+        getInfo: getInfo,
+		getSubInfo: getSubInfo,
```
Comment (suryajak) [inline jazz_cloud-logs-streamer/components/utils.js]: indentation

File: jazz_cloud-logs-streamer/components/utils.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+/**
+    Helper functions for Cloud Logs Streamer
+  @module: utils.js
+  @description: Helper functions for using Regex patterns, etc.
+  @author: SIDD
```
Comment (suryajak) [inline jazz_cloud-logs-streamer/components/utils.js]: Remove author

File: jazz_cloud-logs-streamer/components/utils.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+var isValidJson = function(message) {
+    try {
+        JSON.parse(message);
+    } catch (e) { return false; }
+    return true;
+}
+
+
+var getInfo = function(messages, patternStr) {
+	var pattern = new RegExp(patternStr);
```
Comment (suryajak) [inline jazz_cloud-logs-streamer/components/utils.js]: fix indentation

File: jazz_cloud-logs-streamer/index.js [hunk 4]
```diff
@@ -129,30 +143,78 @@ function transform(payload) {
...
+						data.message = data.message.replace(timestmp,  "");
+
+						var guid = utils.getSubInfo(data.message, globalConfig.PATTERNS.guid_regex, 0);
+						data.message = data.message.replace(guid, "");
+
+						data.message = data.message.replace(data.log_level, "");
+					}
+
+					data.message = data.message.trim();
+
```
Comment (suryajak) [inline jazz_cloud-logs-streamer/index.js]: remove empty lines

File: jazz_cloud-logs-streamer/components/utils.js (hunk 2 above)
Comment (sidddutta) [reply jazz_cloud-logs-streamer/components/utils.js]: Author removed

File: jazz_cloud-logs-streamer/components/utils.js (hunk 3 above)
Comment (sidddutta) [reply jazz_cloud-logs-streamer/components/utils.js]: Indentation and formatting fixed and files

File: jazz_cloud-logs-streamer/index.js (hunk 4 above)
Comment (sidddutta) [reply jazz_cloud-logs-streamer/index.js]: Empty lines removed

**Pull request merged**